src/ui/common/src/components/integrations/addIntegrations.tsx

```diff
@@ -144,43 +158,227 @@ const AddIntegrationListItem: React.FC<AddIntegrationListItemProps> = ({
     </Box>
   );
 
-  return (
-    <Box key={service}>
-      <Box>
-        {iconWrapper}
-        {showDialog && (
-          <IntegrationDialog
-            user={user}
-            service={service}
-            onSuccess={() => {
-              setShowDialog(false);
-              setShowSuccessToast(service);
-            }}
-            onCloseDialog={() => {
-              setShowDialog(false);
-              dispatch(resetConnectNewStatus());
-            }}
-            showMigrationDialog={() => setShowMigrationDialog(true)}
-          />
-        )}
+  if (service !== 'Kubernetes') {
+    return (
+      <Box key={service}>
+        <Box>
+          {iconWrapper}
+          {showDialog && (
+            <IntegrationDialog
+              user={user}
+              service={service}
+              onSuccess={() => {
+                setShowDialog(false);
+                setShowSuccessToast(service);
+              }}
+              onCloseDialog={() => {
+                setShowDialog(false);
+                dispatch(resetConnectNewStatus());
+              }}
+              showMigrationDialog={() => setShowMigrationDialog(true)}
+            />
+          )}
+        </Box>
+        <Snackbar
+          anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+          open={showSuccessToast === service}
+          onClose={handleSuccessToastClose}
+          key={'integrations-dialog-success-snackbar'}
+          autoHideDuration={6000}
+        >
+          <Alert
+            onClose={handleSuccessToastClose}
+            severity="success"
+            sx={{ width: '100%' }}
+          >
+            {`Successfully connected to ${service}!`}
+          </Alert>
+        </Snackbar>
       </Box>
-      <Snackbar
-        anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
-        open={showSuccessToast === service}
-        onClose={handleSuccessToastClose}
-        key={'integrations-dialog-success-snackbar'}
-        autoHideDuration={6000}
-      >
-        <Alert
+    );
+  } else {
+    const handleRegularK8s = () => {
```

Comment:
This Kubernetes dialog should probably go into it's own component. 

After doing so, I'd have the if statement above just check for Kubernetes and render the KubernetesDialog component. No need for an else since you're just returning.


Also, should we choose to use this multistep pattern more often in the future, it might be a good idea to use the Stepper component to show that there are multiple steps to connect.
https://mui.com/material-ui/react-stepper/#horizontal-stepper

Reply:
Something that we want to avoid in the future is checking for the service name and then choosing which component to show. 

We've already got a SupportedIntegrations map which contains the integrations that we support, so we can put the dialogs that we wish to show in there. This work is coming in #1232